app/assets/stylesheets/views/mod-center.scss

```diff
@@ -0,0 +1,143 @@
+@import 'variables';
+@import 'mixins';
```

Comment:
You can drop these two - they are not needed here.

Reply:
For `mod-actions.scss` seems like you gonna need these, but you have to fix paths as well:

```
@import '../variables';
@import '../mixins';
```